python/py-catalogue/Portfile

```diff
@@ -38,7 +38,12 @@ if {${name} ne ${subport}} {
     depends_build-append \
                     port:py${python.version}-setuptools
 
-    depends_run-append \
+    if {${python.version} < "38"} {
```

Comment:
no need for ```"``` around the number, no other Portfile does this

Reply:
Thanks done. 🦃🦃🦃